### PR TITLE
Enable mybinder.org to install the environment

### PR DESCRIPTION
**Pull request by stargaser**
Addresses #106 

* Place files in root directory to make the repo work with mybinder.org
* Add a "launch binder" badge 

TODO:
- [x] Make a Dockerfile in the root directory to be used with Binder
- [x] Test on mybinder.org
- [x] Add the badge

EDIT: Revised the plan

### CODE REVIEW

**Comment by stargaser** [comment]
@adrn did  you have to modify `environment.yml` to make it work on Flatiron Binder?

The build on mybinder.org takes a long time because of Qt which I suppose is brought in by Glue.

**Comment by adrn** [comment]
I didn't modify it, other than creating a link from the top level!

File: 00-Install_and_Setup/environment.yml [hunk 1]
```diff
@@ -31,4 +31,3 @@ dependencies:
     - "glue-vispy-viewers>=0.6"
     - "glueviz>=0.9.1"
     - astroquery
-    - --pre
```
Comment (pllim) [inline 00-Install_and_Setup/environment.yml]: I am not sure about removing the `--pre`. People would be getting the last release, not the latest pre-release. Do we want this?

Comment (stargaser) [reply 00-Install_and_Setup/environment.yml]: I think this is what we don't want. There are a couple of workarounds I saw online. Or we could put a Dockerfile in the root directory for Binder to use.

@adrn does the Flatiron Binder use the repo2docker package? I installed it and it choked on this line.

Comment (bsipocz) [reply 00-Install_and_Setup/environment.yml]: The ``--pre`` was there for astroquery only. I would say we need to keep that on for now.

Comment (bsipocz) [reply 00-Install_and_Setup/environment.yml]: for the binder it'll be OK to pin the version number to be an exact match for a dev version from pypi, but in the docs given for participants ``--pre`` should stay until we come around and change to the new version number scheme

**Comment by pllim** [comment]
Might solve #120

**Comment by stargaser** [comment]
After discussing with @eteq and @pllim we decided to put a Dockerfile in the root directory that will make Binder work, and to document this sufficiently so that future workshop contributors can maintain it.

**Comment by stargaser** [comment]
Using a Dockerfile is complicated but it works now. @adrn does it work at Flatiron?

**Comment by adrn** [comment]
I just checked out your branch in the Flatiron binder and it seems to have used the dockerfile, so I think it worked!

**Pull request merged**